WORKSPACE

```diff
@@ -29,6 +30,7 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #    name="androidndk",
 #    path="<PATH_TO_NDK>",
 #    api_level=14)
+#Note that API level is not the same as NDK version.
```

Comment:
Can you sink to the latest version please and move this line above the newly added NDK comment? It's also missing a space after the #